src/modules/configurator/state/configurationsState.tsx

```diff
@@ -0,0 +1,7 @@
+import { atom } from "recoil";
+import { Configuration } from "../types/configurationsType";
+
+export const userConfigurationsState = atom<Configuration[] | undefined>({
```

Comment:
The initial state can be undefined, but you can have an empty array as initial state, depending on your case.

Reply:
Set default value userConfigurationsState as empty array and removed undefined from type definition.